app/build.gradle.kts

```
@@ -118,8 +129,13 @@ android {
         applicationId = "app.revanced.manager"
         minSdk = 26
         targetSdk = 35
-        versionCode = 1
-        versionName = "0.0.1"
+        versionName = if (version == "unspecified") "1.0.0" else version.toString()
+        versionCode = with(versionName!!.toVersion()) {
```

A little correct semantics. We infer code from version. Also naming the variable "versionName" prevents "version.toVersion()"

Maybe reposition code or accept
```suggestion
        var versionString = if (version == "unspecified") "1.0.0" else version.toString()
        versionName = versionString
        versionCode = with(versionString.toVersion()) {
```